force-app/main/default/classes/DailyIterator_TEST.cls

```diff
@@ -4,19 +4,111 @@ public with sharing class DailyIterator_TEST {
     private static void shouldAssignRule() {
         RecurrenceRule rRule = new RecurrenceRule();
         DailyIterator iterator = new DailyIterator(rRule);
-        System.assertEquals(
-            rRule,
-            iterator.recurrenceRule,
-            'Should assign rule on construction.'
-        );
+        System.assertEquals(rRule, iterator.rRule, 'Should assign rule on construction.');
+    }
+
+    /* FUNCTIONAL TESTS */
+
+    @IsTest
+    private static void shouldIterateEveryDay5Times() {
+        Datetime datetimeStart = Datetime.newInstance(2020, 9, 1, 14, 0, 0); // Sept 1 2020, 2PM
```

Comment:
Docs are inconsistent but I feel like DateTime should have the T capital everywhere, thoughts?